autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/importing/InBrowserPromoPreviousPromptsStore.kt

```diff
@@ -48,10 +56,20 @@ class InMemoryInBrowserPromoPreviousPromptsStore @Inject constructor(
             eTldPlusOne.equals(previousETldPlusOneOffered, ignoreCase = true)
         }
     }
-}
 
-interface InBrowserPromoPreviousPromptsStore {
+    override suspend fun clear() {
+        previousETldPlusOneOffered = null
+    }
+}
 
-    suspend fun recordPromoDisplayed(originalUrl: String)
-    suspend fun hasPromoBeenDisplayed(originalUrl: String): Boolean
+/**
+ * Additional interface for internal/testing use
+ */
+interface InternalInBrowserPromoStore : InBrowserPromoPreviousPromptsStore {
```

Comment:
This internal interface should be in the internal module, is it possible to move?